internal/importers/base/importer.go

```diff
@@ -210,26 +212,28 @@ func (i *Importer) ImportPackages(packages []ImportedPackage, defaultConstraintF
 		if prj.LockHint != "" {
 			var isTag bool
 			// Determine if the lock hint is a revision or tag
-			isTag, version, err = i.isTag(pc.Ident, prj.LockHint)
-			if err != nil {
-				return err
-			}
-
-			// If the hint is a revision, check if it is tagged
-			if !isTag {
-				revision := gps.Revision(prj.LockHint)
-				version, err = i.lookupVersionForLockedProject(pc.Ident, pc.Constraint, revision)
-				if err != nil {
-					version = nil
-					i.Logger.Println(err)
+			if isTag, version, err = i.isTag(pc.Ident, prj.LockHint); err != nil {
+				i.Logger.Printf(
```

Comment:
I was looking to test this and from what I can tell, an error would be returned only if the package did not exist(or network errs but we aren't really testing for that). Perhaps it might be more appropriate to skip the project on encountering an error here?